src/app/components/MediaLoader/index.tsx

```diff
@@ -238,40 +237,50 @@ const MediaLoader = ({ blocks, className, embedded }: Props) => {
           <Metadata blocks={blocks} embedURL={playerConfig?.externalEmbedUrl} />
         )
       }
-      {orientation === 'portrait' && (
+      {showPortraitTitle && (
         <strong css={styles.titlePortrait}>Watch Moments</strong>
       )}
       <figure
         data-e2e="media-loader__container"
-        css={styles.figure}
         className={className}
+        css={[
+          styles.figure(embedded),
+          playerConfig?.ui?.skin === 'classic' && [
+            orientation === 'portrait' && styles.portraitFigure(embedded),
+            orientation === 'landscape' && styles.landscapeFigure,
+          ],
+        ]}
       >
-        {showAds && <AdvertTagLoader />}
-        <BumpLoader />
-        {showPlaceholder ? (
-          <Placeholder
-            src={placeholderConfig?.placeholderSrc}
-            srcSet={placeholderConfig?.placeholderSrcset}
-            noJsMessage={placeholderConfig?.translatedNoJSMessage}
-            mediaInfo={placeholderConfig?.mediaInfo}
-            orientation={orientation}
-            onClick={() => setIsPlaceholder(false)}
+        {isAmp ? (
+          <Amp
+            src={ampIframeUrl}
+            title={mediaInfo?.title}
+            placeholderSrc={placeholderSrc}
+            placeholderSrcset={placeholderSrcset}
+            noJsMessage={translatedNoJSMessage}
           />
         ) : (
-          <MediaContainer
-            playerConfig={playerConfig}
-            showAds={showAds}
-            mediaType={mediaType}
-            orientation={orientation}
-          />
+          <>
+            {showAds && <AdvertTagLoader />}
```

Comment:
This is where the inception of this component starts 😅 

So when on a `.amp` route, it will load the `<Amp />` component in MediaLoader. This will construct the `amp-iframe`, of which the `src` will point to our av-embeds route. Inside the iframe on the av-embeds route, it will then render the _canonical_ MediaLoader, so the ads component will load at that point if its allowed to do so.

Reply:
I've had to take a pause a number of times when working on it. Hard as well as we should try to keep styling of the media player and its parent elements to a minimum as it could then be duplicated in a sense since it renders the component _again_ inside an iframe.